configs/NBD_INFINITYAIO255/config.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F745
+
+#define BOARD_NAME        NBD_INFINITYAIO255
+#define MANUFACTURER_ID   NEBD
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+#define BEEPER_PIN           PD13
+#define MOTOR1_PIN           PC9
+#define MOTOR2_PIN           PC8
+#define MOTOR3_PIN           PC7
+#define MOTOR4_PIN           PC6
+#define LED_STRIP_PIN        PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PB10
+#define UART7_TX_PIN         PE8
+#define UART8_TX_PIN         PE1
+#define UART1_RX_PIN         PB7
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PB11
+#define UART5_RX_PIN         PD2
+#define UART8_RX_PIN         PE0
```

Comment:
Any wire diagram available?

We can assign functions to serial ports if needed when connectors are available.

Reply:
TX: 2, 3, 7, 8
RX: 1, 2, 3, 5, 8

Needs clarification for missing TX1 and TX5 and RX7

Reply:
We will complete the missing UART definition later.

Reply:
> TX: 2, 3, 7, 8 RX: 1, 2, 3, 5, 8
> 
> Needs clarification for missing TX1 and TX5 and RX7

We using UART7 as analog video interface,  so it just using one pin.
And UART5/UART1 as ESC telemetry/SBUS, just need RX.